ui/pages/swaps/prepare-swap-page/index.scss

```diff
@@ -402,3 +402,11 @@
     }
   }
 }
+
+.not-enough-balance {
+  &__banner {}
```

Comment:
Not sure we want to use css if no styles are added

Reply:
This came out of a previous comment from another reviewer: https://github.com/MetaMask/metamask-extension/pull/20241#discussion_r1279378004

Please advise on how you would like to proceed given the TS typing errors that come up.

Reply:
I think you can remove this css, by empty CSS in that comment I meant no css if style is not required. But we can keep the className in the component. 
```suggestion

```